lib/Parse/ParseIfConfig.cpp

```diff
@@ -448,13 +462,16 @@ class EvaluateIfConfigCondition :
       return thisVersion >= Val;
     } else if ((KindName == "swift") || (KindName == "compiler")) {
       auto PUE = cast<PrefixUnaryExpr>(Arg);
+      auto UDRE = getDeclRefStr(PUE->getFn());
```

Comment:
Nitpick: `UDRE` (UnresolvedDeclRefExpr) is a little odd here. I suggest `PrefixName` as per `ValidateIfConfigCondition:: visitCallExpr`.